plugins/lime-plugin-metrics/src/components/box/index.js

```diff
@@ -73,10 +73,12 @@ const Box = ({ station, settings, loading, click, gateway }) => {
 				</b><br /></span>
 			{loading
 				? (<Loading />)
-				: (<div>
-					{station.bandwidth || 0} Mbps / <span>{I18n.t('Packet loss')}</span> {station.loss}%<br />
-					<div style={barStyle(station.loss)} />
-				</div>)
+				: station.bandwidth
+					? (<div>
+						{station.bandwidth || 0} Mbps / <span>{I18n.t('Packet loss')}</span> {station.loss}%<br />
```

Comment:
`{ station.bandwidth || 0 }` This will always resolve to station.bandwith because of previous `?` statement

Reply:
fixed on bf8eb34